index.html

```diff
@@ -107,40 +107,59 @@ <h3>Multi-Post <br> Stories</h3>
+                <li><strong>Languages</strong></li>
+                <li><i class="fa-solid fa-angle-down"></i></li>
             </ul>
-            <button class="resume-mobile-btn" type="submit">Get my resume</button>
-            <ul class="lang-area">
-                <ul class="stacknav">
-                    <li><strong>Languages</strong></li>
-                    <li><i class="fa-solid fa-angle-down"></i></li>
-                </ul>
-                <ul class="stacks">
-                    <li><button class="html" type="submit"><a href="#">HTML</a></button></li>
-                    <li><button class="css" type="submit"><a href="#">CSS</a></button></li>
-                    <li><button class="js" type="submit"><a href="#">JavaScript</a></button></li>
+            <ul class="stacks">
+                <li><button class="html" type="submit"><a href="#">HTML</a></button></li>
+                <li><button class="css" type="submit"><a href="#">CSS</a></button></li>
+                <li><button class="js" type="submit"><a href="#">JavaScript</a></button></li>
+            </ul>
+            <ul>
+                <ul class="frameworks">
+                    <li><strong>Frameworks</strong></li>
+                    <li><i class="fa-solid fa-angle-right"></i></li>
                 </ul>
-                <ul>
-                    <ul class="frameworks">
-                        <li><strong>Frameworks</strong></li>
-                        <li><i class="fa-solid fa-angle-right"></i></li>
-                    </ul>
-                    <ul class="skills">
-                        <li><strong>Skills</strong></li>
-                        <li><i class="fa-solid fa-angle-right"></i></li>
-                    </ul>
+                <ul class="skills">
+                    <li><strong>Skills</strong></li>
+                    <li><i class="fa-solid fa-angle-right"></i></li>
                 </ul>
             </ul>
-        </section>
+        </ul>
+    </section>
+        <!-- contact me  -->
+    <section class="contact">
+        <div class="contact-bg-img">
+            <h6>Contact me</h6>
+            <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I’d love to help with it!</p>
+            <form action="https://formspree.io/f/xjvdador" method="post">
+                <label for="name">Name:</label>
+                <input type="name" id="name" name="user_name" placeholder="Lorem Ipsum" maxlength="30" required>
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="user_name" placeholder="loremipsum@gmail.com" required>
+                <label for="msg">Message:</label>
+                <textarea name="user_name" id="msg" cols="15" rows="10" maxlength="500" required>Write your message here...</textarea>
+                <button class="contact-mobile-btn" type="submit">Get in touch</button>
```

Comment:
- [ ] Great job making the changes requested by the previous reviewer @efmomoh and @Srilearns :clap: However, I noticed that this button doesn't have the right styles applied for the various button interactions that the previous reviewer suggested you include in [this comment](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/pull/3/files#r1259743399). :100: 
- [ ] So kindly ensure that the right styles are applied to your buttons on hover and pressed :wink: 

| Your Design on Hover & Pressed | The Figma Design |
|--------|--------|
| ![image](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/102699958/8fabed8f-de24-4c1a-b41e-4dbf21bc735a) | ![image](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/102699958/850cc102-bf44-45de-86cb-aeb9544eea29)
|

Reply:
- [ ] Good job adding the hover and active styles, however as requested by the previous reviewer, kindly add all the interactions. You are missing the `disabled` and `pressed` effects.

Your CSS
![image](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/81932478/6cec1b36-180b-4edb-a9a9-17ff6d7c00d0)
Check the required change in the original comment [here](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/pull/3/files#r1259743399)